src/Stratis.Bitcoin.Features.Wallet/Wallet.cs

```diff
@@ -180,6 +186,25 @@ public HdAccount AddNewAccount(string password, CoinType coinType, DateTimeOffse
             return accountRoot.AddNewAccount(password, this.EncryptedSeed, this.ChainCode, this.Network, accountCreationTime);
         }
 
+        /// <summary>
```

Comment:
summary is copy pasted from the method above. Parameter name `accountNumber` doesn't have matching xmldoc.

Reply:
nice one